src/ssl/support.cc

```diff
@@ -73,6 +74,43 @@ class OneNameMatcher: public GeneralNameMatcher
     AnyP::Host needle_; ///< a name we are looking for
 };
 
+/// a replacement for CRYPTO_malloc()
+static void *
+CryptoMalloc(size_t num, const char *file, int line)
+{
+    debugs(83, 5, num << " " << file << " " << line);
```

Comment:
Please move this down and report a pointer that was allocated. Same for CryptoFree(). We may even want to add an entry to find-alive.pl to match these.

For CryptoRealloc(), I would probably report as if old memory was deleted and then new memory was allocated (after all the exceptional cases).